### ISSUE TITLE
File containing missing _psl TXT records and registration date < 5 years

### ISSUE THREAD

**Issue by wdhdev** [groundcat/PSL-private-domains-checker#2]
Hey groundcat,

I was wondering if you could add a specific file which shows a list of all the domains where there is no `_psl` TXT record *and* the domain's registration date is less than 5 years ago.

The reason for this is because I believe we could find a bunch of domains that lapsed, and have been re-registered since (potentially because of the PSL status, or the 3rd party didn't realise the domain was on the PSL).

Thanks!

**Comment by groundcat** [comment]
@wdhdev A simple web UI for building and executing SQL queries with customizable conditions is available at https://private-db.publicsuffix.de/. This should be more efficient than working with multiple CSV files based on hard-coded conditions. However, I am currently finding some false positives in the _psl TXT record detection, either due to a bug or inconsistent/unstable DNS queries. I will work on fixing this issue soon.

**Comment by wdhdev** [comment]
Oh, that is pretty good. Do you own that domain, or is it operated elsewhere?

**Comment by groundcat** [comment]
Yes, I have registered this domain specifically for testing purposes and possibly development environments. I chose the '.de' domain extension simply because it was on sale! I have added a disclaimer to prevent any confusion.

**Comment by groundcat** [comment]
Copying @simon-friedberger @dnsguru on this - I created a very basic web interface for executing SQL queries against private section domains at https://private-db.publicsuffix.de/ This should help us more easily identify debris in the list. Please feel free to share any feedback or suggestions! 

This is a temporary tool which should help with **manual** identification of outdated debris, but I do hope that in the future we can find ways to implement some sort of automation. Ideally, the expiration of PSL listings would function similarly to domain name expiration - in an automated and streamlined way.

**Comment by dnsguru** [comment]
helpful - nice work
I think Query 5 may benefit from pulling in the SOA record for review, as it often will be possible to determine the last update (syntax may vary on how the domain admin populates their serial number)

**Comment by fakeboboliu** [comment]
Maybe also cache create date and title of the pull request to another table? It will be easy to figure if the domain is re-registered (for records since migrated to GitHub).
